### PR TITLE
JBIDE-27960 Add EAP 7.4.0.GA in stacks.yaml

### PR DESCRIPTION
**Pull request by jkopriva**
Signed-off-by: Josef Kopriva <jkopriva@redhat.com>

### CODE REVIEW

**Comment by jkopriva** [comment]
@jeffmaury can you please review? Thank you :)

File: stacks.yaml [hunk 1]
```diff
@@ -5827,12 +5852,13 @@ majorReleases:
 
  - name: JBoss EAP
    version: 7
-   recommendedRuntime: *jbosseap720runtime
+   recommendedRuntime: *jbosseap730runtime
```
Comment (jeffmaury) [inline stacks.yaml]: Should be jbosseap740runtime

**Review by jeffmaury — CHANGES_REQUESTED** [review]

File: stacks.yaml (hunk 1 above)
Comment (jkopriva) [reply stacks.yaml]: Thank you! Its changed.

**Review by jeffmaury — APPROVED** [review]
LGTM

**Pull request merged**